src/libs/ReportActionsUtils.ts

```diff
@@ -1437,7 +1441,7 @@ function isCurrentActionUnread(report: OnyxEntry<Report>, reportAction: ReportAc
     if (currentActionIndex === -1) {
         return false;
     }
-    const prevReportAction = sortedReportActions[currentActionIndex - 1];
+    const prevReportAction = sortedReportActions.at(currentActionIndex - 1);
```

Comment:
Make sure that `currentActionIndex - 1` won't be negative, since it can affect the logic then.